src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -141,6 +141,8 @@
       need to be parsed from Maestro++ auto-update and passed through.
     -->
     <EnvironmentVariables Include="OfficialBuildId=$(OfficialBuildId)" />
+    <!-- The BUILD_BUILDNUMBER environmental variable is set by the AzDo agent. We're overwriting it below, so we need to save it in a different env variable -->
+    <EnvironmentVariables Include="VmrBuildNumber=$(BUILD_BUILDNUMBER)"/>
```

Comment:
I think you actually want this in the root `build.proj` and not the repo-project build proj. since the VerticalManifest publish is not specific to a repo project. That probably means you don't have to set an env var, just use BUILD_BUILDNUMBER

Reply:
Btw, I think it should be based off OfficialBuildId. That should be set at some level to BUILD_BUILDNUMBER.

Reply:
I defined the `OfficialBuildId` property in the base `Directory.Build.props`. My msbuild knowledge might be wrong here, but I think it will have the value we want it to when we're calling the `PublishVerticalManifest` task, since the values set in `repo-projects` will be reset

Reply:
Ahh, so this is for future use?

Reply:
No we need it for the `PublishVerticalManifest` task, to set the correct BuildId. But it looks like it's not working as I thought it should. The `OfficialBuildId` is not getting set to anything after we're done with `repo-projs`. Think we'll have to save it in a different property before we start building